e2e/op_stack.go

```diff
@@ -242,6 +243,7 @@ func (op *OPStack) runBatcher(ctx context.Context, env *environment.Env, l1Clien
 	if err := batchSubmitter.StartBatchSubmitting(); err != nil {
 		return fmt.Errorf("start batch submitting: %v", err)
 	}
+
```

Comment:
spurious diff?

Reply:
Ahh.

Some artifact of my local branches intersecting with updated main. I had a debug print in here (see [2nd commit](https://github.com/polymerdao/monomer/pull/68/commits/6bf37ed806ddc3f3680a9509527fc6b080b16bef) on this pr) but it seems it wasn't actually in main.

The net diff seems to be this extra line.